resources/src/operation/mod.rs

```diff
@@ -0,0 +1,220 @@
+/// type of payment starts as one type of asset and ends as another type of asset. There can be
+/// other assets that are traded into and out of along the path.
+#[derive(Debug, Deserialize, Serialize)]
+struct PathPayment {
+    from: String,
+    to: String,
+    destination_asset: AssetIdentifier,
+    destination_amount: Amount,
+    source_asset: AssetIdentifier,
+    source_max: Amount,
+    source_amount: Amount,
+}
+
+/// A “Manage Offer” operation can create, update or delete an offer to trade assets in the Stellar
+/// network. It specifies an issuer, a price and amount of a given asset to buy or sell.
+#[derive(Debug, Deserialize, Serialize)]
+struct ManageOffer {
+    offer_id: i64,
+    selling: AssetIdentifier,
+    buying: AssetIdentifier,
+    amount: Amount,
+    #[serde(rename = "price_r")] price_ratio: PriceRatio,
+    price: Amount,
+}
+
+/// “Create Passive Offer” operation creates an offer that won’t consume a counter offer that
+/// exactly matches this offer. This is useful for offers just used as 1:1 exchanges for path
+/// payments. Use Manage Offer to manage this offer after using this operation to create it.
+#[derive(Debug, Deserialize, Serialize)]
+struct CreatePassiveOffer {
+    offer_id: i64,
+    selling: AssetIdentifier,
+    buying: AssetIdentifier,
+    amount: Amount,
+    #[serde(rename = "price_r")] price_ratio: PriceRatio,
+    price: Amount,
+}
+
+/// Use “Set Options” operation to set following options to your account:
+///
+/// Set/clear account flags:
+/// AUTH_REQUIRED_FLAG (0x1) - if set, TrustLines are created with authorized set to false
+/// requiring the issuer to set it for each TrustLine.
+/// AUTH_REVOCABLE_FLAG (0x2) - if set, the authorized flag in TrustLines can be cleared.
+/// Otherwise, authorization cannot be revoked.
+/// Set the account’s inflation destination.
+/// Add new signers to the account.
+/// Set home domain.
+#[derive(Debug, Deserialize, Serialize)]
+struct SetOptions {
+    signer_key: String,
+    signer_weight: u8,
+    master_key_weight: u8,
+    low_threshold: u32,
+    med_threshold: u32,
+    high_threshold: u32,
+    home_domain: String,
+    set_flags: Option<Vec<u32>>,
+    set_flags_s: Option<Vec<String>>,
+    clear_flags: Option<Vec<u32>>,
+    clear_flags_s: Option<Vec<String>>,
+}
+
+/// Use “Change Trust” operation to create/update/delete a trust line from the source account to
+/// another. The issuer being trusted and the asset code are in the given Asset object.
+#[derive(Debug, Deserialize, Serialize)]
+struct ChangeTrust {
+    asset: AssetIdentifier,
+    trustee: String,
+    trustor: String,
+    limit: Amount,
+}
+
+/// Removes the account and transfers all remaining XLM to the destination account.
+#[derive(Debug, Deserialize, Serialize)]
+struct AccountMerge {
+    account: String,
+    into: String,
+}
+
+/// Runs inflation
+#[derive(Debug, Deserialize, Serialize)]
+struct Inflation {}
```

Comment:
Can't the `Inflation` enum value simply not have an inner value?

Reply:
You are correct.